RNA-seq/03-gastric_cancer_exploratory.Rmd

```diff
@@ -166,10 +134,10 @@ ggplot2::ggsave(pca_plot_file, plot = ggplot2::last_plot())
 We don't have batch information (i.e., when the samples were run) for this 
 particular experiment, but let's imagine that `SRR585574` and `SRR585576` were
 run separately from all other samples.
-We'll put this in new "toy" column in `sample_meta_df`.
+We'll put this in new "toy" column in `gene_summarized`.
 
 ```{r batches}
-sample_meta_df$batch <- c("batch1", "batch1", "batch1", "batch1", "batch2", 
+gene_summarized@colData$batch <- c("batch1", "batch1", "batch1", "batch1", "batch2", 
```

Comment:
Also feels a tad clumsy but I don't want to add this in the previous notebook since its fake data.